test/libsolidity/SolidityNameAndTypeResolution.cpp

```diff
@@ -947,6 +947,21 @@ BOOST_AUTO_TEST_CASE(base_constructor_arguments_override)
 	CHECK_SUCCESS(text);
 }
 
+BOOST_AUTO_TEST_CASE(duplicated_super_constructor_call)
+{
+	char const* text = R"(
+		contract A { function A(uint) public { } }
+		contract B is A(2) { function B() A(3) public {  } }
+	)";
+	CHECK_ERROR(text, DeclarationError, "Duplicated super constructor call.");
+
+	text = R"(
+		contract A { function A(uint) public { } }
+		contract B is A { function B() A() public {  } }
+	)"
+	CHECK_SUCCESS(text);
```

Comment:
Shouldn't this be an error too?

Reply:
Yes, I should remove the declared parameter in A. I took the inheritance specifier as only declaring the base contract, not providing arguments to its constructor.

Do you interpret them differently?

Reply:
NVM, I had missed your earlier comments about it.